src/qt/intro.cpp

```diff
@@ -109,52 +107,49 @@ void FreespaceChecker::check()
     Q_EMIT reply(replyStatus, replyMessage, freeBytesAvailable);
 }
 
+namespace {
```

Comment:
Why do we need a namespace here?

Reply:
[Use an unnamed (anonymous) namespace for all internal/nonexported entities](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnonexported-entities)

Reply:
Why has it no intends?

Reply:
> Why has it no intends?

Sorry. "intends" or "indents"?

Reply:
indents, sorry

Reply:
It follows our [Coding Style](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c). Also see [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy).